js/liquid.js

```diff
@@ -210,6 +210,7 @@ module.exports = class liquid extends Exchange {
                 'price_too_high': InvalidOrder,
                 'price_too_small': InvalidOrder, // {"errors":{"order":["price_too_small"]}}
                 'product_disabled': BadSymbol, // {"errors":{"order":["product_disabled"]}}
+                'is not in your IP whitelist': AuthenticationError, // {"message":"93.177.130.181 is not in your IP whitelist"}
```

Comment:
It won't work, cause the error handler for Liquid only calls `throwExactlyMatchedException`, there's no `broad` exception mapping here yet, the `broad` mapping has to be addressed first to make it work with a partial string-error match and use `throwBroadlyMatchedException` for it.

Reply:
ah,  i've not looked at it. obvious,